bayes/src/main/java/com/enigmastation/ml/bayes/Feature.java

```diff
@@ -48,11 +63,39 @@ public Integer getCountForCategory(Object category) {
         return count;
     }
 
+    /**
+     * TODO: Needs to be done
+     *
+     * @param category
+     */
     public void incrementCategoryCount(Object category) {
         Integer oldCount = categories.get(category);
         if (oldCount == null) {
             oldCount = 0;
         }
         categories.put(category, oldCount + 1);
     }
+
```

Comment:
This is also gross. We'd be better off changing the features to be Serializable (and therefore "featureSerializable?" ... no, that's just as gross) than by adding in this rubbish. This was originally part of a Gigaspaces aspect.